Update install notes (#749)
CrayLabs/SmartSim: doc/installation_instructions/troubleshooting/troubleshooting.rst

.. _installation-troubleshooting:

Installation Troubleshooting
============================

> SmartSim has been installed on a variety of systems and our users often have

Reviewer: ```suggestion
SmartSim has been installed on a variety of systems with 
```

different build environments and toolchains. The following two sections detail

> some common situations and how to setup and modify your build environment:

Reviewer: ```suggestion
some common situations and how to configure your build environment:
```

trying to avoid saying `A and B` if `C` encompasses both
